CTOR-921 ! (#3600)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-firewalls-arkoon-snmp.md

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="Net-FW-Arkoon-SNMP-custom" label="Net-FW-Arkoon-SNMP-custom">

| Service Alias | Service Template                 | Service Description         |
|:--------------|:---------------------------------|:----------------------------|
| Load          | Net-FW-Arkoon-Load-SNMP-custom   | Check CPU usage             |
| Memory        | Net-FW-Arkoon-Memory-SNMP-custom | Check memory usage          |
| Swap          | Net-FW-Arkoon-Swap-SNMP-custom   | Check virtual memory usage  |

> The services listed above are created automatically when the **Net-FW-Arkoon-SNMP-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias              | Service Template                                     | Service Description                                             | Discovery  |
|:---------------------------|:-----------------------------------------------------|:----------------------------------------------------------------|:----------:|

> | Packet-Errors-Generic-Name | Net-FW-Arkoon-Packet-Errors-Generic-Name-SNMP-custom | Check packets on errors/discards of a network interface         |            |

Reviewer: ```suggestion
| Packet-Errors-Generic-Name | Net-FW-Arkoon-Packet-Errors-Generic-Name-SNMP-custom | Check packets in error/discarded from a network interface          |            |
```

> | Packet-Errors-Global       | Net-FW-Arkoon-Packet-Errors-Global-SNMP-custom       | Check packets on errors/discards of multiple network interfaces | X          |

Reviewer: ```suggestion
| Packet-Errors-Global       | Net-FW-Arkoon-Packet-Errors-Global-SNMP-custom       | Check packets in error/discarded from multiple network interfaces | X          |
```

> | Process-Generic            | Net-FW-Arkoon-Process-Generic-SNMP-custom            | Check Arkoon process/service is working                         |            |

Reviewer: ```suggestion
| Process-Generic            | Net-FW-Arkoon-Process-Generic-SNMP-custom            | Check that an Arkoon process/service is working                         |            |
```
